docs/pages/blog/2021-q1-update.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Q1 2021 Update
+description: An update on our mission for Q1 2021.
+date: 2021-04-12T00:00:00.000Z
+authors: ['oliviertassinari']
+card: true
+---
+
+This update covers our progress over the last three months and what we aim to achieve in the coming months.
+
+## Product
+
+Here are the most significant improvements since December 2020.
+
+### Design system
+
+- 👩‍🎤 We have worked on rolling out and scaling the new **styling engine** to all the components.
+  At this point, we have migrated 121 components out of 167 to emotion/styled-components.
+  Most of the components left to be migrated are in the lab.
+  We are almost done! You can subscribe to [material-ui#24405](https://github.com/mui-org/material-ui/issues/24405) to be notified once finished.
+  The community has been a valuable aid.
+- 📚 As a corollary to the migration of the components, we have worked on the migration of the **documentation**.
+  We have favoured the `sx` prop anytime possible.
+  When the examples need to adapt to the provided props, we have used the `styled` API.
```

Comment:
"need to adapt to the provided props"

What does this mean?

Reply:
```suggestion
  When a component needs to render styles based on the provided props, we have used the `styled` API.
```